java/app/build.gradle.kts

```diff
@@ -7,7 +7,7 @@ import org.gradle.api.JavaVersion.VERSION_1_8
 
 plugins {
     // Apply the org.jetbrains.kotlin.jvm Plugin to add support for Kotlin.
-    id("org.jetbrains.kotlin.jvm") version "1.8.10"
+    id("org.jetbrains.kotlin.jvm") version "1.5.10"
```

Comment:
Is this really no problem?

Reply:
I think there's no problem because it's built without any warning or errors.

I think the initial version(1.8.10) was from command `gradle init` by gradle installed in the local host.
We can downgrade kotlin version if proto-generated kotlin doesn't require upper kotlin version.